lib/checkout_sdk/instruments/instruments_client.rb

```diff
@@ -2,27 +2,21 @@
 
 module CheckoutSdk
   module Instruments
-    class InstrumentsClient < Client
-      INSTRUMENTS = 'instruments'
+    class InstrumentsClient < CheckoutSdk::Previous::Instruments::InstrumentsClient
```

Comment:
The concept is good, but this should be on opposite way.
Default is the single source of truth, in the future if we remove CS1, this could cause refactors, so this class should be 

`class InstrumentsClient < Client`

and then 

lib/checkout_sdk/instruments/previous//instruments_client.rb

`class InstrumentsClient < CheckoutSdk::Instruments::InstrumentsClient`